genai-perf/genai_perf/parser.py

```diff
@@ -1233,7 +1233,7 @@ def parse_args():
             # this subcommand is deprecated and is not supported in the config file
             return args, None, argv[passthrough_index + 1 :]
         elif args.subcommand == ic.Subcommand.PROCESS.value:
-            config = ConfigCommand()
+            config = ConfigCommand(skip_inferencing_and_checking=True)
             config = add_cli_options_to_config(config, args)
```

Comment:
After the options are added to the config you then need to `config.infer_and_check_options()`.